econml/policy/_forest/_tree.py

```diff
@@ -261,6 +261,30 @@ def predict(self, X, check_input=True):
         pred = self.tree_.predict(X)
         return np.argmax(pred, axis=1)
 
+    def predict_proba(self, X, check_input=True):
+        """ Predict the probability of recommending each treatment
+
+        Parameters
+        ----------
+        X : {array-like} of shape (n_samples, n_features)
+            The input samples. Internally, it will be converted to
+            ``dtype=np.float64``.
+        check_input : bool, default=True
+            Allow to bypass several input checking.
+            Don't use this parameter unless you know what you do.
+
+        Returns
+        -------
+        treatment_proba : array-like of shape (n_samples, n_treatments)
+            The probability of each treatment recommendation
+        """
+        check_is_fitted(self)
+        X = self._validate_X_predict(X, check_input)
+        pred = self.tree_.predict(X)
+        proba = np.zeros(pred.shape)
+        proba[np.arange(X.shape[0]), np.argmax(pred, axis=1)] = 1

```

Comment:
Here and everywhere else argmax is used, do we care at all about what happens in the case of ties?

Reply:
nope. we need to return on. So we could just defer to what argmax does.